File: is_divisible.rb
# def is_divisible(n, x, y)
#   puts (n % x == 0 && n % y == 0) ? 'true' : 'false'
# end

def is_divisible(n, x, y)
  return (n % x == 0 && n % y == 0) ? true : false
end

is_divisible(3, 3, 4)
is_divisible(12, 3, 4)

File: isDivisible.js
// function isDivisible(n, x, y) {
//   if ((n % x === 0) && (n % y === 0)) {
//     console.log(true);
//     // return true;
//   } else {
//     console.log(false);
//     // return false;
//   }
// }

function isDivisible(n, x, y) {
  ((n % x === 0) && (n % y === 0)) ? console.log(true) : console.log(false);
}

isDivisible(3, 3, 4);
isDivisible(12, 3, 4);
